src/main/app/src/app/zaken/zaak-view/zaak-view.component.ts

```diff
@@ -568,20 +569,32 @@ export class ZaakViewComponent extends ActionsViewComponent implements OnInit, A
     }
 
     private openZaakAfsluitenDialog(): void {
-        const dialogData = new DialogData([
-                new InputFormFieldBuilder().id('reden').label('actie.zaak.afsluiten.reden').validators(Validators.required).maxlength(100).build()],
-            (results: any[]) => this.zakenService.afsluiten(this.zaak.uuid, results['reden']).pipe(
-                tap(() => this.websocketService.suspendListener(this.zaakListener))
-            ));
-
-        dialogData.confirmButtonActionKey = 'actie.zaak.afsluiten';
-
-        this.dialog.open(DialogComponent, {data: dialogData}).afterClosed().subscribe(result => {
-            if (result) {
-                this.updateZaak();
-                this.loadTaken();
-                this.utilService.openSnackbar('msg.zaak.afgesloten');
-            }
+        this.planItemsService.getUserEventListenerPlanItemToelichting(this.zaak.uuid, UserEventListenerActie.ZaakAfhandelen)
```

Comment:
Zaak afsluiten is niet gelijk aan Zaak afhandelen. Bij Zaak afsluiten hoeft niet de toelichting getoond worden van zaak afhandelen. Het ophalen van de toelichting hoeft dus niet en de code rondom het ophalen van de toelichting kan dan ook worden verwijderd.